src/main/java/net/fabricmc/loom/providers/MinecraftMappedProvider.java

```diff
@@ -59,24 +65,36 @@ public void provide(DependencyInfo dependency, Project project, LoomGradleExtens
 			if (getIntermediaryJar().exists()) {
 				getIntermediaryJar().delete();
 			}
+
 			new MapJarsTiny().mapJars(minecraftProvider, this, project);
 		}
 
 		if (!MINECRAFT_MAPPED_JAR.exists()) {
 			throw new RuntimeException("mapped jar not found");
 		}
 
-		String version = minecraftProvider.minecraftVersion + "-mapped-" + extension.getMappingsProvider().mappingsName + "-" + extension.getMappingsProvider().mappingsVersion;
-		project.getDependencies().add(Constants.MINECRAFT_NAMED, project.getDependencies().module("net.minecraft:minecraft:" + version));
-		version = minecraftProvider.minecraftVersion + "-intermediary-" + extension.getMappingsProvider().mappingsName;
-		project.getDependencies().add(Constants.MINECRAFT_INTERMEDIARY, project.getDependencies().module("net.minecraft:minecraft:" + version));
+		String mappingsName = extension.getMappingsProvider().mappingsName;
```

Comment:
You no longer use `extension.getMappingsProvider().mappingsVersion` anywhere

Reply:
It's used in `LoomDependencyManager` from what I can see

Reply:
No this is needed as now the jar doesnt have the mappings version. `minecraft-19w45a-mapped-net.fabricmc.yarn-v2.jar` isnt right.

Reply:
Oh! I missed some of the line while copy pasting.